docs/getting_started.rst

```diff
@@ -402,17 +402,198 @@ or `regions.PixelRegion.contains` methods:
+    >>> center = PixCoord(4., 5.)
+    >>> reg = CirclePixelRegion(center, 2.3411)
+    >>> mask = reg.to_mask()
+    >>> mask
+    <regions.mask.Mask object at 0x10900b5c0>
+    >>> mask.data
+    array([[ 0.,  0.,  0.,  0.,  0.,  0.,  0.],
+           [ 0.,  0.,  1.,  1.,  1.,  0.,  0.],
+           [ 0.,  1.,  1.,  1.,  1.,  1.,  0.],
+           [ 0.,  1.,  1.,  1.,  1.,  1.,  0.],
+           [ 0.,  1.,  1.,  1.,  1.,  1.,  0.],
+           [ 0.,  0.,  1.,  1.,  1.,  0.,  0.],
+           [ 0.,  0.,  0.,  0.,  0.,  0.,  0.]])
+
+The mask data contains floating point that are between 0 (no overlap) and 1
+(overlap). By default, this is determined by looking only at the central position
+in each pixel, and::
+
+    >>> reg.to_mask()
+
+is equivalent to::
+
+    >>> reg.to_mask(mode='center')
+
+but other modes are available:
+
+* ``mode='exact'``: the overlap is determined using the exact geometrical
+  overlap between pixels and the region. This is slower than using the central
+  position, but allows partial overlap to be treated correctly.
+
+* ``mode='subpixels'``: the overlap is determined by sub-sampling the pixel
+  using a grid of sub-pixels. The number of sub-pixels to use in this mode
+  should be given using the ``subpixels`` argument.
+
+Here are what the different modes look like:
+
+.. plot::
+   :include-source:
+
+    import matplotlib.pyplot as plt
+    from regions.core import PixCoord
+    from regions.shapes.circle import CirclePixelRegion
+
+    center = PixCoord(6.6, 7.2)
+    reg = CirclePixelRegion(center, 5.2)
+
+    plt.figure(figsize=(6, 6))
+
+    mask1 = reg.to_mask(mode='center')
+    plt.subplot(2, 2, 1)
+    plt.title("mode='center'", size=9)
+    plt.imshow(mask1.data, cmap=plt.cm.viridis,
+               interpolation='nearest', origin='lower')
+
+    mask2 = reg.to_mask(mode='exact')
+    plt.subplot(2, 2, 2)
+    plt.title("mode='exact'", size=9)
+    plt.imshow(mask2.data, cmap=plt.cm.viridis,
+               interpolation='nearest', origin='lower')
+
+    mask3 = reg.to_mask(mode='subpixels', subpixels=3)
+    plt.subplot(2, 2, 3)
+    plt.title("mode='subpixels', subpixels=3", size=9)
+    plt.imshow(mask3.data, cmap=plt.cm.viridis,
+               interpolation='nearest', origin='lower')
+
+    mask4 = reg.to_mask(mode='subpixels', subpixels=20)
+    plt.subplot(2, 2, 4)
+    plt.title("mode='subpixels', subpixels=20", size=9)
+    plt.imshow(mask4.data, cmap=plt.cm.viridis,
+               interpolation='nearest', origin='lower')
+
+As we've seen above, the :class:`~regions.Mask` objects have a ``data``
+attribute that contains a Numpy array with the mask values. However, if you
+have for example a circular region with a radius of 3 pixels at a pixel position
+of (1000, 1000), it would be inefficient to store a mask that has a size larger
+than this, so instead we store the mask using the minimal array that contains
+the mask, and the :class:`~regions.Mask` objects also include an ``origin``
```

Comment:
This needs to be updated, no?
You're saying from the high-level docs that mask has an origin attribute, but it now has a `bbox` attribute.
Also - please link to the `regions.BoundingBox` class at least once from the high-level docs.
And maybe link back from the class docstring to this section, or add one example there.